cmd/svcat/instance/touch_cmd.go

```diff
@@ -56,6 +58,27 @@ func (c *touchInstanceCmd) Validate(args []string) error {
 }
 
 func (c *touchInstanceCmd) Run() error {
+	instance, err := c.App.RetrieveInstance(c.Namespace, c.name)
```

Comment:
Oops! 😊 This isn't what I mean by having touch reuse the set logic. Would you please back this out and we can sync up and do that in a separate issue and PR after this is merged?

I think that the entire command could use a refresh to improve how it's written (it was an older command), so I'll make an issue with ideas for how it could be implemented.